packages/cuba-react-ui/src/ui/table/DataTableHelpers.tsx

```diff
@@ -49,6 +53,7 @@ export interface DataColumnConfig {
  *
  * @param config
  *
+ * todo it's better to move example in separate .tsx class for better readability and IDE type check
```

Comment:
I think we should handle it as part of #110, we should probably have an Examples section at our future documentation site, compile-check for all examples, etc.